### DISCUSSION TITLE
Alexa su seconda istanza HA

### DISCUSSION THREAD

**Discussion by vincenzocoletta** [indomus/forum#2946] [GENERALE - Discussioni generiche]
Ciao, qualcuno ha esperienza di integrazione di Alexa in HA per due istanze indipendenti? Sulla prima ho già effettuato le integrazioni via haaska e AWS grazie alle preziose guide di InDomus ma ora dovrei integrare la stessa Alexa (ho un solo account Amazon) sulla seconda istanza HA; devo replicare tutta la guida (nuovo profilo di sicurezza, nuova skill, nuova funzione AWS Lambda) oppure alcuni dei passaggi possono essere riutilizzati? Grazie in anticipo a chiunque riuscisse a fornirmi indicazioni o spunti!

**Comment by snakuzzo** [comment]
Devi replicare

**Reply by vincenzocoletta** [reply]
Tks! Mi metto all'opera e vi aggiorno se dovessi trovare qualche scorciatoia...

**Reply by vincenzocoletta** [reply]
Confermo, tutto molto semplice seguendo le guide ma segnalo soltanto due step che volendo possono essere evitati (magari chiari ai più esperti, un pò meno ai neofiti come me). Il **Security Profile**: può essere utilizzato quello creato per configurare la prima istanza e il **Ruolo** in AWS-IAM: anche in questo caso il ruolo **_lambda_basic_execution_** già creato in precedenza può essere riutilizzato nella creazione di una nuova funzione **_haaska2_** che poi utilizzerà la nuova skill (anche in questo caso una nuova **_haaska2_**).

**Reply by snakuzzo** [reply]
Sì, corretto,
